### DISCUSSION TITLE
How do I use the guild search endpoint?

### DISCUSSION THREAD

**Discussion by Vresod** [Rapptz/discord.py#9142] [Q&A] [answered]
The discord API supports searching guilds for messages. This is done using the `/channels/{channel_id}/messages/search?content={search}` endpoint. I was unable to find a way to use this feature in the library; if anyone could point me in the right direction that'd be great. Thank you!

**Comment by lmaotrigine** [comment, marked as answer]
hello
bots do not have access to the search endpoint.

**Reply by Vresod** [reply]
Damn ok thanks

**Reply by uukelele** [reply]
No, they do (at least now)

**Reply by uukelele** [reply]
https://github.com/discord/discord-api-docs/discussions/3216#discussioncomment-14147090
